packages/database/src/migrations/20210728020128-AddFQSSurveySyncGroups-modifies-data.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+import { generateId, insertObject, deleteObject } from '../utilities';
+import {
+  FQS1_CODE_MAPPING,
+  FQS2_CODE_MAPPING,
+} from './migrationData/20210726233011-AddFQSQuestionMappings';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const KOBO_SURVEYS = [
+  {
+    code: 'FQS1',
+    entityQuestionCode: 'FQS1Primary_location/FQS1PrimarySchool',
+    questionCodeMapping: FQS1_CODE_MAPPING,
+  },
+  {
+    code: 'FQS2',
+    entityQuestionCode: 'FQS2Primary_location/FQS2PrimarySchool',
+    questionCodeMapping: FQS2_CODE_MAPPING,
+  },
+];
+
+exports.up = async function (db) {
+  for (const { code, entityQuestionCode, questionCodeMapping } of KOBO_SURVEYS) {
+    await insertObject(db, 'data_service_sync_group', {
+      id: generateId(),
+      code,
+      service_type: 'kobo',
+      config: {
+        // koboSurveyCode: 'xyz',
```

Comment:
Figured I'd keep these codes as a manual step. Feels like something that shouldn't be committed to the repo

Reply:
Hmm I would be pretty comfortable committing it to the repo, it seems to be in the same class of info as the question code mapping. Makes release process easier!